pkgs/development/python-modules/cfn-lint/default.nix

```diff
@@ -64,8 +58,8 @@ buildPythonPackage rec {
     "cfnlint.transform"
   ];
 
-  checkInputs = [ pytestCheckHook mock pydot ];
-  preCheck = "export PATH=$out/bin:$PATH";
+  # checkPhase collects 0 tests, then fails after running the empty test suite.
```

Comment:
Please add at least a pythonImportsCheck. Also why did you switch away from GitHub releases? They probably include the test suite.

Reply:
The test suite tried to execute `git grep` to attempt to build documentation or something, which failed because `fetchFromGitHub` does not give you a git repo.

From there I assumed that pypi was a better source for built releases than an in-development repository.

Reply:
If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.

Reply:
https://github.com/NixOS/nixpkgs/pull/105658/files#diff-ee6b773ddf3418a3eb1b48c3c80b07326ec36ed4258812381196e408123996c7R45-R59

This already exists. Is it turned off by `doCheck = false`?

Reply:
pythonImportsCheck is not turned of by doCheck and makes sure the package is at least importable and the requires modules are there.

Reply:
pythonImportsCheck is not turned of by doCheck.